cargo-dist-schema/src/lib.rs

```diff
@@ -106,7 +106,6 @@ pub struct DistManifest {
     pub linkage: Vec<Linkage>,
     /// Files to upload
```

Comment:
do you think we may want to leave a comment here about why we want to serialize the empty array?

Reply:
Good callout, done.